src/Symfony/Component/Messenger/Command/DebugCommand.php

```diff
@@ -113,4 +121,20 @@ private function formatConditions(array $options): string
 
         return ' (when '.implode(', ', $optionsMapping).')';
     }
+
+    private static function getClassDescription(string $class): string
```

Comment:
Note this is borrowed from `\Symfony\Bundle\FrameworkBundle\Console\Descriptor\Descriptor::getClassDescription`, to avoid requiring a dependency to the fwb.

Reply:
what about moving it up to the base `Descriptor`class in the console component?

Reply:
It'll require the config component as well for the extra checks in the original method (which I'm not sure would make sense here) + require `symfony/console:^5.1`
I saw fit to start with this simplified version as private method for this need, at least as a start.

Reply:
Fine for me then, thought it was copy-pasted.